cmd/telegraf/telegraf.go

```diff
@@ -71,6 +72,7 @@ func (t *Telegraf) Init(pprofErr <-chan error, f Filters, g GlobalFlags, w Windo
 	t.pprofErr = pprofErr
 	t.inputFilters = f.input
 	t.outputFilters = f.output
+	t.configFiles = []string{}
```

Comment:
```suggestion
```

Not necessary as the default value of a slice is a zero-length slice

Reply:
Updated